server/src/main/java/org/spine3/server/entity/EntityRepository.java

```diff
@@ -146,23 +148,22 @@ public E find(I id) {
      * @return all the entities in this repository with the IDs matching the given {@code Iterable}
      */
     @CheckReturnValue
-    public ImmutableCollection<E> findBulk(Iterable<I> ids) {
+    public ImmutableCollection<E> findBulk(Iterable<I> ids, @Nullable FieldMask fieldMask) {
```

Comment:
As we discussed previously, please provide both `findBulk(ids)` and `findBulk(ids, fieldMask)` signatures.